epub_translator/llm/core.py

```
@@ -16,6 +18,89 @@
+        executor: LLMExecutor,
+        cache_path: Path | None,
+    ) -> None:
+        self._executor = executor
+        self._cache_path = cache_path
+        self._context_id = uuid.uuid4().hex[:12]
+        self._temp_files: list[Path] = []
+
+    def __enter__(self) -> Self:
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb) -> None:  # noqa: ANN001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `ANN001`)

Remove unused `noqa` directive

(RUF100)